src/controllers/accommodation.controller.js

```diff
@@ -42,6 +42,23 @@ class AccommodationController {
     ResponseService.setSuccess(200, 'list of available accommodations', availableAccommodations);
     return ResponseService.send(res);
   }
+
+  /**
+   * @static
+   * @param {*} req
+   * @param {*} res
+   * @returns {*} created Accomodation
+   * @memberof AccommodationController
+   */
+  static async createAccomodation(req, res) {
+    const newAccommodation = await AccommodationService.createAccomodationWithInclude(req.body);
+    const getTotalRooms = await AccommodationService.GetTotalRooms();
+    const totalRooms = getTotalRooms.filter((key) => key.accommodationId === newAccommodation.id);
+
```

Comment:
What's the difference between line 55 and 56?

Reply:
Line 55 I am getting a sum of all room in our database and on line 56 I am selecting only the room for the accommodation we just created.

Reply:
I have realized that when I created an accommodation the fields total rooms and available rooms are not available they were just being added in the response not the accommodations table. So that's why I am adding them.